src/chrome/content/rules/Disqus.com.xml

```diff
@@ -1,52 +1,32 @@
 <!--
 	Commenting system used by several large sites.
 
-
 	Nonfunctional domains:
-
-		disqus.com subdomains:
-
-			- blog		(tumblr)
-			- community     (tumblr)
-
-	Fully covered subdomains:
-
-		- go
-		- referrer
-		- juggler.services
-		- realtime.services
-
+		- blog.disqus.com
+		- community.disqus.com
+		
+	Subdomains like SITENAME.disqus.com aren't covered in this ruleset.
+	Probably some other subdomains aren't covered either.
 -->
-<ruleset name="Disqus (unreliable)" default_off="https://www.eff.org/r.5abU">
+<ruleset name="Disqus (partial)">
 
 	<target host="disqus.com" />
-	<target host="*.disqus.com" />
-		<!--
-			https://trac.torproject.org/projects/tor/ticket/5496
-
-			https://mail1.eff.org/pipermail/https-everywhere-rules/2012-May/001176.html
-
-		<exclusion pattern="^http://disqus\.com/next/lounge/client\.html" />
-		<exclusion pattern="^http://\w+\.disqus\.com/\d+/build/system/embed\.js"/>
-
-			See comments at top.
-							-->
-		<exclusion pattern="^http://(?:blog|community)\.disqus\.com/" />
-
-
-	<rule from="^http://(?:www\.)?disqus\.com/"
```

Comment:
This rule used to strip the 'www' from www.disqus.com. Have you confirmed that's not necessary anymore?